front/components/assistant/AssistantDetails.tsx

```diff
@@ -140,7 +140,7 @@ function AssistantDetailsPerformance({
 
   return (
     <>
-      <div className="flex flex-row justify-between gap-3">
+      <div className="flex flex-row items-center justify-between gap-3">
```

Comment:
alignment between Analytics and dropdown

before:
<img width="574" alt="Screenshot 2025-05-14 at 21 27 52" src="https://github.com/user-attachments/assets/8c7521ea-6464-42cd-838a-50ad7bdaba9c" />

ater:
<img width="574" alt="Screenshot 2025-05-14 at 21 28 11" src="https://github.com/user-attachments/assets/d41cd4c0-e721-4dcd-87ff-b41dc97c36cc" />